src/views/Home/components/MilestoneCard/MilestoneCard.tsx

```diff
@@ -69,6 +73,29 @@ const MilestoneCard: FC<MilestoneCardProps> = ({ slug, milestoneData }) => {
           <ProgressLabel progress={progress}>{usLocalizedNumber(progress * 100, 0)}%</ProgressLabel>
         </ProgressBarContainer>
       </Progress>
+      <CoordinatorsContainer
+        ref={(element) => {
+          if (element !== null) {
+            coordinatorsRef.current.push(element);
```

Comment:
If we push the element here, we could be adding the element multiple times on every render

Reply:
This definitely needs to be done very carefully... I've changed the approach to adjust sections heights to a simpler more "pure js" solution based on class names. This can be improved in the future...